registration/app/services/NotificationRegistrarSupport.scala

```diff
@@ -0,0 +1,19 @@
+package services
+
+import javax.inject.Inject
+
+import notifications.providers.{NotificationRegistrar, WindowsNotificationProvider}
+import play.api.libs.ws.WSClient
+
+import scala.concurrent.ExecutionContext
+
+final class NotificationRegistrarSupport @Inject()(wsClient: WSClient, configuration: Configuration)(implicit executionContext: ExecutionContext) {
+
+  val providerConf = configuration.notificationHubConfiguration.fold(error => throw new Exception(error.message), identity)
```

Comment:
I don't think we should expose it as public, controllers shouldn't have an access here